File: catorce ejercicios/PYTHON2/ejercicio.5.py
"""Los empleados de una fábrica trabajan en dos turnos: diurno y nocturno. Se desea calcular el
jornal diario de acuerdo con los siguientes puntos:
La tarifa de las horas diurnas es de 10 dólares.
La tarifa de las horas nocturnas es de 15 dólares.
Codifique un algoritmo que solicite el turno de trabajo y el número de horas trabajadas y
posteriormente calcule el valor a pagar."""


turno = input ("Ingrese su turno de trabajo (Diurno o Nocturno): ")
horas = int(input ("Ingrese las horas: "))

turno = turno.lower()
if turno == "diurno":
    resultado = horas * 10
    print ("El valor a pagar es de: ",resultado," dolares")
elif turno == "nocturno":
    resultado2 = horas * 15
    print ("El valor a pagar es de:",resultado2, " dolares")
else:
    print("ERROR")

File: catorce ejercicios/PHP2/Php/ejercicio.5.php
<!-- Los empleados de una fábrica trabajan en dos turnos: diurno y nocturno. Se desea calcular el
jornal diario de acuerdo con los siguientes puntos:
La tarifa de las horas diurnas es de 10 dólares.
La tarifa de las horas nocturnas es de 15 dólares.
Codifique un algoritmo que solicite el turno de trabajo y el número de horas trabajadas y
posteriormente calcule el valor a pagar. -->


<form method="POST" action="">
        <label for="turno">Ingrese el turno de trabajo (diurno/nocturno):</label>
        <input type="text" id="turno" name="turno" required><br>
        
        <label for="horas">Ingrese el número de horas trabajadas:</label>
        <input type="number" id="horas" name="horas" required><br>
        
        <input type="submit" value="Calcular">
    </form>

<?php
if ($_SERVER["REQUEST_METHOD"] === "POST") {
    $turno = strtolower($_POST['turno']);
    $horas = $_POST['horas'];

    $Diurna = 10;
    $Nocturna = 15;

    $valorAPagar = 0;

    if ($turno === "diurno") {
        $valorAPagar = $horas * $Diurna;
        $respuesta = "El total a pagar es de: " . $valorAPagar . " dólares.";
    } else if ($turno === "nocturno") {
        $valorAPagar = $horas * $Nocturna;
        $respuesta = "El total a pagar es de: " . $valorAPagar . " dólares.";
    } else {
        $respuesta = "El turno ingresado no es válido.";
    }
    echo $respuesta;
}
?>
